### ISSUE TITLE
Changing maximum height of paper input.

### ISSUE THREAD

**Issue by ruifung** [PolymerElements/paper-input#372]
### Description

Setting the height via css of paper input causes it to act wierdly.
### Expected outcome
The paper input element shrinks to fit.

### Actual outcome
the paper-container shrinks, but everything else stays in the same position

### Browsers Affected

- [ ] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by notwaldorf** [comment]
Yeah, `paper-input-container` is a fairly big element, that has a lot of styles and nested elements (such as the label and the input). Just changing its height will probably not propagate as you would expect to all of its children elements.

**Comment by cnbuff410** [comment]
What's the suggested workaround in this case?

**Comment by cormacguerin** [comment]
Yeah I was also unable to work out a good way to change the height of paper-input.
Any ideas?

**Comment by powelljordan** [comment]
Bump
